conda/recipes/librmm/recipe.yaml

```
@@ -72,40 +74,72 @@ outputs:
       ignore_run_exports:
         by_name:
           - cuda-version
+          - if: cuda_major == "11"
+            then: cudatoolkit
+          - if: linux
+            then:
+              - __glibc
+              - libgcc
+              - libstdcxx
+              - if: cuda_major == "11"
+                then:
+                  - libgcc-ng
+                  - libstdcxx-ng
+                  - sysroot_linux-64
     tests:
       - script:
           - "test -d \"${PREFIX}/include/rmm\""
     about:
       homepage: ${{ load_from_file("python/librmm/pyproject.toml").project.urls.Homepage }}
       license: ${{ load_from_file("python/librmm/pyproject.toml").project.license.text }}
       summary: ${{ load_from_file("python/librmm/pyproject.toml").project.description }}
+
   - package:
       name: librmm-tests
       version: ${{ version }}
     build:
       string: cuda${{ cuda_major }}_${{ date_string }}_${{ head_rev }}
-      script:
-        - cmake --install build --component testing
+      script: cmake --install build --component testing
+      dynamic_linking:
+        overlinking_behavior: "error"
+        missing_dso_allowlist:
+          - "libbenchmark.so*"
     requirements:
       build:
+        - ${{ compiler('c') }}
```

Why only C and not C++/CUDA? Also double-quotes.
```suggestion
        - ${{ compiler("c") }}
```

---

I was just squashing overlinking stuff, but I think it's better to have all the compilers in `build`